### ISSUE TITLE
Player Reported Bugs (mangos one)

### ISSUE THREAD

**Issue by Mibu86** [CoronaCore/Issues#225]
u should rlly take a look here...
http://forum.wowgate.eu/index.php?/forum/112-bug-243-tbc/
So many priest issues are being reported
Lol i just saw u there :P

**Comment by Salja** [comment]
Yes but pls give me more details about vampiric touch i dont understand he say missing tooltip but tooltip is client side
